### PR TITLE
refactor: Apply HTML plugin directly in webpack.config.js

### PR DESCRIPTION
**Pull request by jahredhope**
Apply `html-render-webpack-plugin` directly in Webpack config.
This API became available with `v1.1.0` and removes the need to apply the plugin seperately.

This won't result in any behaviour changes.

### CODE REVIEW

**Comment by mattsjones** [comment]
Can we make this a `refactor` rather then a `fix`. If there is zero behaviour difference we shouldn't perform a release.

File: test/test-cases/custom-src-paths/.gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # managed by sku
 .eslintrc
 .prettierrc
+coverage/
```
Comment (mattsjones) [inline test/test-cases/custom-src-paths/.gitignore]: Realise this has nothing to do with your PR but pretty weird it's showing in this PR.

Comment (markdalgleish) [reply test/test-cases/custom-src-paths/.gitignore]: It's because the Renovate PR didn't run `sku configure`.

Comment (jahredhope) [reply test/test-cases/custom-src-paths/.gitignore]: This will go away once I rebase off the changes from https://github.com/seek-oss/sku/pull/253/

Comment (jahredhope) [reply test/test-cases/custom-src-paths/.gitignore]: This is now gone with merge from master

**Pull request merged**